### PR TITLE
Integrated loans to invest with getline.ts

### PR DESCRIPTION
**Pull request by balinskia**
Removed mocks and now fundraising loans are fetched from blockchain

### CODE REVIEW

File: frontend/src/api/invest.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import API, { LoanState } from './index';
+import { LoanToInvestT } from '@/store/invest/types';
+import { getTokenSymbolsFromBlockchain, getAmountsWantedFromBlockchain, getAmountsGatheredFromBlockchain } from './utils';
+import { getSingleAmountWantedFromBlockchain, getSingleAmountGatheredFromBlockchain, getSingleTokenSymbolFromBlockchain } from './utils';
+
+
+export async function getLoansToInvest(cb) {
+  const api = await API.instance();
+  let currentUser = await api.currentUser();
```
Comment (q3k) [inline frontend/src/api/invest.ts]: `const`

File: frontend/src/api/invest.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  let blockchainLoans = await api.loansByState(LoanState.Fundraising);
```
Comment (q3k) [inline frontend/src/api/invest.ts]: `const`

File: frontend/src/api/invest.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { BigNumber } from 'bignumber.js';
+
+import API, { LoanState } from './index';
+import { LoanToInvestT } from '@/store/invest/types';
+import { getTokenSymbolsFromBlockchain, getAmountsWantedFromBlockchain, getAmountsGatheredFromBlockchain } from './utils';
+import { getSingleAmountWantedFromBlockchain, getSingleAmountGatheredFromBlockchain, getSingleTokenSymbolFromBlockchain } from './utils';
+
+
+export async function getLoansToInvest(cb) {
```
Comment (q3k) [inline frontend/src/api/invest.ts]: `cb: (LoansToInvestT[]): void`

File: frontend/src/api/invest.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  cb(loansToInvest);
+}
+
+export async function getLoanToInvest(shortId) {
+  const api = await API.instance();
+  let currentUser = await api.currentUser();
+  let blockchainLoan = await api.loan(shortId);
+  await blockchainLoan.updateStateFromBlockchain();
+
+  const amountWanted: BigNumber = await getSingleAmountWantedFromBlockchain(blockchainLoan);
```
Comment (q3k) [inline frontend/src/api/invest.ts]: It seems like we always download those three at the same time - maybe add a `get{Single/Many}LoanFromBlockchain` that calls all three in parallel via `Promise.all`?

File: frontend/src/components/common/UserScore.vue [hunk 5]
```diff
@@ -1,26 +1,39 @@
...
+import { goToBlogspotScore } from '@/router/redirects';
+
+/*
+ * Right now it always diplays "?" score
+ * In future we will have the score in props
+ */
 export default {
   name: 'UserScore',
-  props: ['value'],
+  // props: ['value'],
```
Comment (q3k) [inline frontend/src/components/common/UserScore.vue]: Don't commit commented code.

File: frontend/src/api/invest.ts (hunk 4 above)
Comment (balinskia) [reply frontend/src/api/invest.ts]: I think that concatenation of lists: ```amountWanted```, ```amountsGathered``` and ```tokenSymbols``` into one list and than doing ```Promise.all``` will of course be a little bit faster, but also very hard to understand

**Pull request merged**